modules/consensus/persist.go

```diff
@@ -25,7 +25,15 @@ func (cs *ConsensusSet) initSetDB() error {
 	if err != nil {
 		return err
 	}
-	// Explicit initilization preferred to implicit
+
+	// Update the siafundoutput diffs map for the genesis block
+	// on disk
+	cs.updateDatabase = true
+	for _, sfod := range cs.blockRoot.SiafundOutputDiffs {
+		cs.commitSiafundOutputDiff(sfod, modules.DiffApply)
+	}
```

Comment:
I am confused about what is happening here. Why are you committing to the diff set in `persist.go`?